go/tools/bzltestutil/BUILD.bazel

```diff
@@ -3,17 +3,15 @@ load("//go:def.bzl", "go_test", "go_tool_library")
 go_tool_library(
     name = "bzltestutil",
     srcs = [
-        "init.go",
         "lcov.go",
         "test2json.go",
         "wrap.go",
         "xml.go",
     ],
-    # We add "+initfirst/" to the package path so this package is initialized
-    # before user code. See comment above the init function in init.go.
     importmap = "+initfirst/github.com/bazelbuild/rules_go/go/tools/bzltestutil",
```

Comment:
Do we still need the funky importpath on this package?

Reply:
Nope, deleted.